Isssue with missing dependency 'lodash.isnan' in '/node_modules/@react-pdf/core/lib/elements'

Hey guys, could you please help me with this error

../~/@react-pdf/core/lib/elements/Text.js
Module not found: Can't resolve 'lodash.isnan' in '/node_modules/@react-pdf/core/lib/elements'

I created a react app using creat-react-app and added following:
`yarn add @react-pdf/core`
`yarn add @react-pdf/node`

After that, I copy pasted the sample code given on git (https://github.com/diegomura/react-pdf)

Do I have to install this missing dependency?

--- a/examples/package.json
+++ b/examples/package.json
@@ -9,6 +9,7 @@
   "dependencies": {
     "@react-pdf/core": "^0.2.0",
     "@react-pdf/node": "^0.2.0",
+    "lodash.times": "^4.3.2",
     "react": "16.0.0-alpha.4"
   }
 }

--- a/package.json
+++ b/package.json
@@ -49,8 +49,6 @@
     "fbjs": "^0.8.12",
     "jest": "^20.0.3",
     "lerna": "2.0.0-rc.3",
-    "lodash.isnan": "^3.0.2",
-    "lodash.times": "^4.3.2",
     "opencollective": "^1.0.3",
     "request": "^2.81.0"
   },

--- a/packages/react-pdf/package.json
+++ b/packages/react-pdf/package.json
@@ -24,6 +24,7 @@
     "fontkit": "^1.5.4",
     "hex-rgb": "^1.0.0",
     "lodash.isfunction": "^3.0.8",
+    "lodash.isnan": "^3.0.2",
     "lodash.map": "^4.6.0",
     "lodash.topairsin": "^4.3.0",
     "lodash.upperfirst": "^4.3.1",

--- a/packages/react-pdf/src/elements/Text.js
+++ b/packages/react-pdf/src/elements/Text.js
@@ -1,7 +1,6 @@
 import Base from './Base';
 import Yoga from 'yoga-layout';
-import isNan from 'lodash.isnan';
-import upperFirst from 'lodash.upperfirst';
+import { isNan, upperFirst } from 'lodash';
 
 class Text extends Base {
   width = null;

